client/components/telecom/telephony/group/number/feature/ovhPabx/menu/edit/telephony-group-number-feature-ovh-pabx-menu-edit.html

```diff
@@ -200,42 +200,38 @@ <h2 data-ng-if="['greetSound', 'invalidSound', 'greetSoundTts', 'invalidSoundTts
-                                           data-ng-value="'none'"
-                                           data-ng-change="$ctrl.onSoudTypeChange()" />
-                                </flat-radio>
-                            </div>
-                            <div class="dtd vertical-middle">
-                                <span class="block"
-                                      data-translate="telephony_number_feature_ovh_pabx_menu_options_sound_type_none">
-                                </span>
-                            </div>
+                        <div class="d-table-cell valign-middle radio-content">
+                            <flat-radio>
+                                <input type="radio"
+                                       name="soundTypeChoice"
+                                       data-ng-model="$ctrl.model[$ctrl.menuCtrl.popoverStatus.rightPage === 'greetSoundType' ? 'greetSoundType' : 'invalidSoundType']"
+                                       data-ng-value="'none'"
+                                       data-ng-change="$ctrl.onSoudTypeChange()" />
+                            </flat-radio>
+                        </div>
+                        <div class="d-table-cell align-middle">
+                            <span class="block"
+                                  data-translate="telephony_number_feature_ovh_pabx_menu_options_sound_type_none">
+                            </span>
                         </div>
                     </label>
-                    <label class="section-item item-choice full-width pointer dtable"
+                    <label class="section-item item-choice w-100 dtable"
                            data-ng-repeat="soundType in $ctrl.availableSoundTypes track by $index">
-                        <div class="dtr">
-                            <div class="dtd vertical-middle radio-content">
-                                <flat-radio>
-                                    <input type="radio"
-                                           name="soundTypeChoice"
-                                           data-ng-model="$ctrl.model[$ctrl.menuCtrl.popoverStatus.rightPage === 'greetSoundType' ? 'greetSoundType' : 'invalidSoundType']"
-                                           data-ng-value="soundType"
-                                           data-ng-change="$ctrl.onSoudTypeChange()" />
-                                </flat-radio>
-                            </div>
-                            <div class="dtd vertical-middle">
-                                <span class="block"
-                                      data-translate="{{ 'telephony_number_feature_ovh_pabx_menu_options_sound_type_' + soundType }}">
-                                </span>
-                            </div>
+                        <div class="d-table-cell align-middle radio-content">
+                            <flat-radio>
```

Comment:
We have to remove `<flat-radio>` directive.

Reply:
In a future time

Reply:
No, remove it please, it's one of v1 objectives.

We should clean flat input element and use ui-kit or ui-kit-bs to homogenize the base code

Reply:
In an another pull-request